tests/ops/qutrit/test_qutrit_parametric_ops.py

```diff
@@ -0,0 +1,331 @@
+            qml.TRX.compute_matrix(np.pi / 2, subspace=[1, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(np.pi / 2, wires=0, subspace=[1, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        expected = np.array([[1, 0, -1j], [0, np.sqrt(2), 0], [-1j, 0, 1]]) / np.sqrt(2)
+        assert np.allclose(
+            qml.TRX.compute_matrix(np.pi / 2, subspace=[0, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(np.pi / 2, wires=0, subspace=[0, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        # test identity for broadcasted theta=pi/2
+        pi_half = np.array([np.pi / 2] * 2)
+        expected = np.tensordot(
+            [1, 1], np.array([[1, -1j, 0], [-1j, 1, 0], [0, 0, np.sqrt(2)]]) / np.sqrt(2), axes=0
+        )
+        assert np.allclose(
+            qml.TRX.compute_matrix(pi_half, subspace=[0, 1]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(pi_half, wires=0, subspace=[0, 1]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        expected = np.tensordot(
+            [1, 1], np.array([[1, 0, -1j], [0, np.sqrt(2), 0], [-1j, 0, 1]]) / np.sqrt(2), axes=0
+        )
+        assert np.allclose(
+            qml.TRX.compute_matrix(pi_half, subspace=[0, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(pi_half, wires=0, subspace=[0, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        expected = np.tensordot(
+            [1, 1], np.array([[np.sqrt(2), 0, 0], [0, 1, -1j], [0, -1j, 1]]) / np.sqrt(2), axes=0
+        )
+        assert np.allclose(
+            qml.TRX.compute_matrix(pi_half, subspace=[1, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(pi_half, wires=0, subspace=[1, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        # test identity for theta=pi
+        expected = -1j * np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1j]])
+        assert np.allclose(
+            qml.TRX.compute_matrix(np.pi, subspace=[0, 1]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(np.pi, wires=0, subspace=[0, 1]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        expected = -1j * np.array([[1j, 0, 0], [0, 0, 1], [0, 1, 0]])
+        assert np.allclose(
+            qml.TRX.compute_matrix(np.pi, subspace=[1, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(np.pi, wires=0, subspace=[1, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+        expected = -1j * np.array([[0, 0, 1], [0, 1j, 0], [1, 0, 0]])
+        assert np.allclose(
+            qml.TRX.compute_matrix(np.pi, subspace=[0, 2]), expected, atol=tol, rtol=0
+        )
+        assert np.allclose(
+            qml.TRX(np.pi, wires=0, subspace=[0, 2]).matrix(), expected, atol=tol, rtol=0
+        )
+
+
+# TODO: Add tests for grad
```

Comment:
Is it possible add these yet? It would be good to have tests for all the interfaces to ensure the numerical values are being handled as expected, both in terms of matrix construction as well as gradients.

Reply:
I've looked into the tests for testing grad in `tests/ops/qubit/test_parametric_ops.py`, and those tests are almost entirely focused on gradient testing of parametrized operations other than the rotations, so I'm not sure if these tests are required or not. Let me know what you think.

Reply:
Oh I see, those are for very specific operations. I did a bit of digging and it looks like  `gradients/test_parameter_shift.py` contains tests the Pauli rotations, controlled rotations, etc. Could you please take a look at those, and see if it makes sense to add something there?

Reply:
Agreed, we should validate that the computed gradients are in fact correct, irrespective of method 👍🏼

Reply:
I see now the parameter frequencies and shift rule are being tested, but is the actual value of a gradient (from, e.g., a circuit which measures a Gell-Mann expectation value) actually being evaluated and tested somewhere?